modules/openapi-generator-cli/src/test/resources/batch/jaxrs-datelib-j8-nested-include.json

```diff
@@ -0,0 +1,10 @@
+{
+  "!include": "common/jaxrs-datelib-j8.json",
+  "generatorName": "jaxrs-jersey",
+  "inputSpec": "batch/specs/petstore.yaml",
```

Comment:
I wonder if we should move `inputSpec` from here to `jaxrs-datelib-j8-nested.json` to verify that the initial `!include` processing will also load sibling properties?

I tested the suggestion locally for both JSON and YAML to verify the functionality.